beacon_node/http_api/src/lib.rs

```diff
@@ -892,10 +892,34 @@ pub fn serve<T: BeaconChainTypes>(
 
                     match chain.process_block(block.clone()) {
                         Ok(root) => {
+                            let attestation_deadline = chain.slot_clock.unagg_attestation_production_delay();
+                            if delay >= attestation_deadline {
```

Comment:
There is logic almost identical to this a few lines down in this file (line 944) with `crit_threshold`, maybe we could tweak that instead?

Reply:
Oh you're totally right! Glad we already have this logic! I'll fix this :)